lib/ClimaLandSimulations/Manifest.toml

```diff
@@ -0,0 +1,2658 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
did you add this file on purpose?

Reply:
let me try removing it, and Ill add something to the docs, and then merge on Monday